src/main/java/space/obminyashka/items_exchange/service/impl/SendGridService.java

```diff
@@ -51,7 +55,14 @@ public void sendMail(String emailTo, EmailType subject, Locale locale) throws IO
 
     @Override
     public void validateEmail(UUID validationCode) throws EmailValidationCodeNotFoundException, EmailValidationCodeExpiredException {
-        //TODO in ticket OBMIN-344
+        Optional<EmailConfirmationCode> optionalEmail = emailRepository.findById(validationCode);
```

Comment:
What loading type LAZY or EAGER would that request produce?

Reply:
@Sershok it would be better if you answer the question or just posted "it will be changed/updated" as a response